code/backend/pages/02_Explore_Data.py

```diff
@@ -38,13 +38,21 @@
 try:
     search_handler = Search.get_search_handler(env_helper)
 
-    results = search_handler.search_client.search("*", facets=["title"])
-    unique_files = [filename["value"] for filename in results.get_facets()["title"]]
+    results = (
```

Comment:
we have a `search_handler`, yet we expose the `search_client` and let people call search. I feel we have a leaky abstraction here that could be tidied up. Will also keep the streamlit code simple. Happy for it to be a seperate PR if you want to create one (if others agree!)

Reply:
+1. Also this if-else will not be needed here if we stop exposing the search_client directly and handle it from the search_handler directly.

Reply:
Created separate method for this and removed the exposed search_client from Explore_Data page